backend/app/src/test/resources/application.yaml

```diff
@@ -45,8 +45,8 @@ tolgee:
     signingSecret: fakeSecret
   postgres-autostart:
     enabled: true
-    container-name: tolgee_backend_tests_postgres_main
-    port: 55433
+    container-name: tolgee_backend_tests_postgres_main_server_app
```

Comment:
Why this change?

Reply:
I am spawning separate docker containers for app and ee. Which resolves the conflicts when running the tests in parallel, and improves the performance, As it seemed it halved the time for running all the tests in my desktop (project drive is not on SSD)

Reply:
OK, makes sense!